source/exec/docker.d

```diff
@@ -79,10 +93,15 @@ class Docker: IExecProvider
 			--queueSize_;
 
 		auto encoded = Base64.encode(cast(ubyte[])source);
+		// try to find the compiler in the available images
+		auto r = DockerImages.find!(d => d.canFind(compiler));
```

Comment:
It's actually intended (we are going to fix the compilers the user can select anyways) and I wanted to have a simple solution that works without allocation.